bin/functions.sh

```diff
@@ -1,12 +1,7 @@
 function startConjur() {
-  local conjurType="$1"
   local services='conjur cuke-master'
 
-  if [[ "$conjurType" == "oss" ]]; then
-    services='conjur'
-  elif [[ "$conjurType" == "enterprise" ]]; then
-    services='cuke-master'
-  fi
+  services='conjur'
```

Comment:
Should the services variable not just be set to `conjur` immediately then ?

Reply:
Oh yeah :D

Reply:
Fixed